hardcoded/hardcoded.go

```diff
@@ -1,17 +1,16 @@
-// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
Did the version of go-bindata change? I don't see a corresponding change in go.mod.

Reply:
It did not. 

But: https://github.com/go-bindata/go-bindata/commit/6518908443d2a120be39ea957b64ac051e8f1fbd introduced that change a while ago... no idea why we are just seeing that now.

Reply:
I think this may not be intended, my guess is that a `go-bindata` other than the one installed by `make install_deps` is being used.